pkg/build/netbsd.go

```diff
@@ -66,3 +73,47 @@ func (ctx netbsd) clean(kernelDir string) error {
 	// machine.
 	return nil
 }
+
+// Copy the compiled kernel to the qemu disk image using ssh
+func CopyKernelToDisk(outputDir string) error {
+	temp := []byte(`
+{
+	"qemu": "qemu-system-x86_64",
+	"cpu": 1,
+	"count": 1,
+	"mem": 1024
+}	`)
+	VMconfig := (*json.RawMessage)(&temp)
+	// Create config for booting the disk image
+	cfg := &mgrconfig.Config{
+		Workdir:      outputDir,
+		Image:        filepath.Join(outputDir, "image"),
+		SSHKey:       filepath.Join(outputDir, "key"),
+		SSHUser:      "root",
+		TargetOS:     "netbsd",
+		TargetVMArch: "amd64",
+		Type:         "qemu",
+		VM:           *VMconfig,
+	}
+	// Create a VM pool
+	pool, err := vm.Create(cfg, true)
+	if err != nil {
+		return fmt.Errorf("Failed to create a VM Pool : %v", err)
+	}
+	// Create a VM instance (We need only one)
+	inst, err := pool.Create(0)
+	if err != nil {
+		return fmt.Errorf("Failed to create the VM Instance : %v", err)
+	}
+	defer inst.Close()
+	// Copy the kernel into the disk image and replace it
+	kernel, err := inst.Copy(filepath.Join(outputDir,"netbsd"))
```

Comment:
Add a comment here saying that we are relying on the fact that vm/qemu copies everything into root dir for netbsd. Strictly saying it's not guaranteed.

Reply:
Hmm.. I thought the archconfig in vm/qemu/qemu.go decided that part. The target dir for netbsd is currently '/' so why would that be a problem?

Reply:
It's not a problem currently. But it may become a problem tomorrow, day after tomorrow or any other day. Nothing will stop somebody from changing "/" to "/tmp" or something else and this build will break later with obscure manifestation and somebody may spend significant time figuring out what happens. It's better to make it fail loud, but I am not sure what's a good way to do it (any ideas?), so at least leaving a comment to somebody who will spent days starring at this code and trying to figure out what happens will be useful.